src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,94 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Storage {
+    public static final String SERIALIZATION_PATH = "data/task_list.txt";
+
+    public static void updateLocalFile(TaskList taskList) throws IOException {
+
+        ArrayList<Task> list = taskList.arrayList;
```

Comment:
```suggestion
        ArrayList<Task> tasks = taskList.arrayList;
```
Perhaps you could name it in plural form? It may also be better to give it a name that informs other developers about what the variable contains